apps/prairielearn/src/sync/fromDisk/assessments.js

```diff
@@ -268,6 +270,21 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
   return assessmentParams;
 }
 
+function parseSharedQuestionReference(qid) {
+  const firstSlash = qid.indexOf('/');
+  if (firstSlash === -1) {
+    return {
+      sharing_name: qid.substring(1, qid.length),
+      qid: '',
+    };
+  } else {
+    return {
+      sharing_name: qid.substring(1, firstSlash),
+      qid: qid.substring(firstSlash + 1, qid.length),
+    };
+  }
+}
```

Comment:
I'm trying to understand what this does. When would we hit the first case? As in, is `@cs225` a valid question reference? It can't be... right? Or am I misinterpreting what this function does?

Reply:
Ya, that's right, it's always an error case. Obviously when the database is queried a few lines of code later, nothing will match the empty QID. Should we change this somehow to make it more obvious that it's an error case?

Reply:
Fair, ok. I agree that we'll end up with an error in that case. I was thinking maybe this would be better to have in `checkAndRecordQid` in the other file, but then we'd still have to deal with parsing over here, and we'd still get the second error.

Reply:
I went ahead and added a comment here to make sure its clear this is an error case